Procfile

```diff
@@ -1,3 +1,2 @@
-release: bundle exec rails db:migrate
-release: bundle exec rails csv:load
+release: bash ./release-tasks.sh
```

Comment:
I don't think the `bash` part is needed here as there is a hash bang in the file which determines how the file should be run.